packages/fast-components-react-base/src/tabs/tabs.props.ts

```diff
@@ -5,6 +5,22 @@ import {
 } from "@microsoft/fast-components-class-name-contracts-base";
 import { Orientation } from "@microsoft/fast-web-utilities";
 
+export interface TabConfig {
+    className: string;
+    key: string;
+    "aria-controls": string;
+    active: boolean;
+    onClick: (e: React.MouseEvent) => void;
+    onKeyDown: (e: React.KeyboardEvent) => void;
+    tabIndex: number;
+}
+
+export interface TabItems {
```

Comment:
this is the interface for a single TabItem, there should be no "s"

Reply:
Then we have a name conflict with TabItem which already exists. Remembering now... I think this is why I choose the name tabItemData